requirements.txt

```diff
@@ -14,12 +14,11 @@ click>=6.6 # BSD
 click-spinner>=0.1.6 # MIT
 click-repl>=0.1.0 # MIT
 asciitree>=0.3.3 # MIT
-terminaltables>=3.0.0 #  MIT
+tabulate>=0.8.2 # MIT
```

Comment:
This change also needs to be done in the new `minimum-constraints.txt` file added with PR #100.

Reply:
DONE.